packages/dialtone-css/lib/build/less/recipes/emoji_row.less

```diff
@@ -9,7 +9,29 @@
 }
 
 .d-recipe-emoji-row__tooltip-content {
-   max-width: var(--dt-size-975);
+   max-inline-size: var(--dt-size-875);
+}
+
+.d-recipe-emoji-row__tooltip-emoji {
+  display: inline-block;
+  padding: var(--dt-space-300);
+  background-color: var(--dt-color-neutral-white);
```

Comment:
do we want this to be always white or should it be dark in dark mode? It looks a bit inconsistent. Our emoji are dark mode compatible so I think it should be fine to make dark.